Makefile

```diff
@@ -1,4 +1,3 @@
-BOARD ?= arduino_101
```

Comment:
It maybe a good idea to leave as is in case they use just Zephyr to build and not define BOARD